logos/benchmarks/benchmark_logos.py

```
@@ -864,6 +864,52 @@ def _parse_float_or_none(raw: Optional[str]) -> Optional[float]:
+# on-wire send time. _raise_fd_limit() (called from main) ensures the process has
+# enough file descriptors for the resulting socket count.
+_HTTP_LIMITS = httpx.Limits(max_connections=None, max_keepalive_connections=50)
+
+
+def _build_load_client(timeout_s: float) -> httpx.AsyncClient:
+    """AsyncClient for the load runners: uncapped connection pool and an explicit
+    timeout with ``pool=None`` so a dispatch never blocks waiting for a pool slot.
+    Read/write/connect still honour ``timeout_s``. With no pool wait, the sent_at
+    captured just before ``client.stream`` is the true moment the request goes out."""
+    timeout = httpx.Timeout(timeout_s, pool=None)
+    return httpx.AsyncClient(timeout=timeout, limits=_HTTP_LIMITS, verify=False)
```

Ruff (0.15.17)

[error] Probable use of `httpx` call with `verify=False` disabling SSL certificate checks

(S501)